templates/triliumnext.xml

```diff
@@ -0,0 +1,24 @@
+<?xml version="1.0"?>
+<Container version="2">
+  <Name>triliumnext</Name>
+  <Repository>triliumnext/notes:latest</Repository>
+  <Registry>https://hub.docker.com/r/triliumnext/notes</Registry>
+  <Network>bridge</Network>
+  <Shell>sh</Shell>
+  <Privileged>false</Privileged>
+  <Support>https://forums.unraid.net/topic/87798-support-selfhostersnets-template-repository/</Support>
+  <Project>https://github.com/TriliumNext/Notes</Project>
+  <Overview>TriliumNext Notes is an open-source, cross-platform hierarchical note taking application with focus on building large personal knowledge bases.</Overview>
+  <Category>Productivity: Tools:Utilities</Category>
+  <WebUI>http://[IP]:[PORT:8080]</WebUI>
+  <TemplateURL>https://raw.githubusercontent.com/selfhosters/unRAID-CA-templates/master/templates/triliumnext.xml</TemplateURL>
+  <Icon>https://raw.githubusercontent.com/selfhosters/unRAID-CA-templates/master/templates/img/triliumnext.png</Icon>
+  <Description>TriliumNext Notes is a hierarchical note taking application with focus on building large personal knowledge bases.</Description>
+  <Networking>
```

Comment:
Remove this tag, unneeded since `Network` is defined above

Reply:
Removed, also spotted an empty Labels tag I missed the first time around, removed that as well.